spark/pom.xml

```diff
@@ -0,0 +1,344 @@
+			<artifactId>accumulo-core</artifactId>
+			<version>${accumulo.version}</version>
+			<exclusions>
+				<!-- This exclusion needs to be made in order for unit tests that run 
+					MapReduce jobs over MockAccumulo instances to run. We exclude it here and 
+					include guava 14.0.1 below (see JIRA ACCUMULO-3100 which applies this patch 
+					for Accumulo 1.6.1). -->
+				<exclusion>
+					<groupId>com.google.guava</groupId>
+					<artifactId>guava</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>com.azavea.geotrellis</groupId>
+			<artifactId>geotrellis-spark_2.10</artifactId>
+			<version>0.10.0-M1</version>
+			<exclusions>
+				<!-- We only need a couple of classes from this package so exclude most 
+					of its dependencies. -->
+				<exclusion>
+					<groupId>com.azavea.geotrellis</groupId>
+					<artifactId>geotrellis-raster_2.10</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.azavea.geotrellis</groupId>
+					<artifactId>geotrellis-gdal_2.10</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.azavea.geotrellis</groupId>
+					<artifactId>geotrellis-index_2.10</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.apache.accumulo</groupId>
+					<artifactId>accumulo-core</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.apache.spark</groupId>
+					<artifactId>spark-core_2.10</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.apache.hadoop</groupId>
+					<artifactId>hadoop-client</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.quantifind</groupId>
+					<artifactId>sumac_2.10</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>de.javakaffee</groupId>
+					<artifactId>kryo-serializers</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.google.uzaygezen</groupId>
+					<artifactId>uzaygezen-core</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.amazonaws</groupId>
+					<artifactId>aws-java-sdk-s3</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.spire-math</groupId>
+					<artifactId>spire_2.10</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.github.julien-truffaut</groupId>
+					<artifactId>monocle-core_2.10</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.github.julien-truffaut</groupId>
+					<artifactId>monocle-macro_2.10</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.github.nscala-time</groupId>
+					<artifactId>nscala-time_2.10</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.scalaz.stream</groupId>
+					<artifactId>scalaz-stream_2.10</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.gdal</groupId>
+					<artifactId>gdal</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+
+		<!-- Test -->
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.11</version>
```

Comment:
In the gaffer parent pom we include junit version 4.12 can we just use that instead? If so, this junit dependency can be removed.